### PR TITLE
fix:minimize-help-code(vac-1333)

### PR DESCRIPTION
**Pull request by Msubhashini1**
**Description** : Implemented minimized mode for the help component.
Test Steps: 1. Log in into the console app.
                     2. From left menu click on help icon, (i.e) last Icon  from bottom.
                     3. Help window will be open.
                     4. Inside window there will be maximized icon.
                     5. After we clicking on the maximized icon windo
Minimized 2022-05-16 at 1 50 20 PM
w will be minimized and the icon will be changed to maximized. (vice versa will happen)

Maximized 2022-05-16 at 1 50 34 PM

                     

#### Test Steps

- [x] `npm run start`
- [ ] then this
- [ ] finally this

#### General Tests for Every PR

- [x] `npm run start` still works.
- [x] `npm run lint` passes.
- [x] `npm run stylelint` passes.
- [x] `npm test` passes and code coverage is not lower.
- [x] `npm run build` still works.

### CODE REVIEW

**Comment by owilliams320** [comment]
Can you include a description of what is changing here and test steps? also a screen shot

**Comment by Msubhashini1** [comment]
>

**Comment by Msubhashini1** [comment]
> Can you include a description of what is changing here and test steps? also a screen shot

**Description** : Implemented minimized mode for the help component.
Test Steps: 1. Log in into the console app.
                     2. From left menu click on help icon, (i.e) last Icon  from bottom.
                     3. Help window will be open.
                     4. Inside window there will be maximized icon.
                     5. After we clicking on the maximized icon windo
Minimized 2022-05-16 at 1 50 20 PM
w will be minimized and the icon will be changed to maximized. (vice versa will happen)

Maximized 2022-05-16 at 1 50 34 PM

**Review by owilliams320 — CHANGES_REQUESTED** [review]
looks good for the most part but looks like you still need to adjust the positioning when the help menu is minimized

Screen Shot 2022-05-18 at 4 36 40 PM

### Correct Spec
Screen Shot 2022-05-18 at 4 38 00 PM

**Comment by adamnel** [comment]
There should be no gap between the icon buttons (the close icon and the dock icon). Right now looks like the markup has a space/line return in it which is separating them more than they should be.

Also, please verify in dark theme if you haven't already.

**Comment by Msubhashini1** [comment]
> looks good for the most part but looks like you still need to adjust the positioning when the help menu is minimized
> 
> Screen Shot 2022-05-18 at 4 36 40 PM
> 
> ### Correct Spec
> Screen Shot 2022-05-18 at 4 38 00 PM

@owilliams320 As per comments adjusted the positioning of the help menu.


Minimized 2022-05-23 at 2 20 32 PM

**Comment by Msubhashini1** [comment]
> There should be no gap between the icon buttons (the close icon and the dock icon). Right now looks like the markup has a space/line return in it which is separating them more than they should be.
> 
> Also, please verify in dark theme if you haven't already.

@adamnel Removed space between the Icons of closed and minimized.
Screen Shot 2022-05-23 at 2 20 03 PM
Screen Shot 2022-05-23 at 2 20 20 PM

File: libs/markdown-navigator/src/markdown-navigator-window-service/markdown-navigator-window.service.ts [hunk 1]
```diff
@@ -42,8 +42,8 @@ export interface IMarkdownNavigatorWindowConfig {
 
 const CDK_OVERLAY_CUSTOM_CLASS = 'td-window-dialog';
 
-const DEFAULT_POSITION: DialogPosition = { bottom: '0px', right: '0px' };
-const DEFAULT_WIDTH = '360px';
+const DEFAULT_POSITION: DialogPosition = { bottom: '0px', right: '10px' };
```
Comment (owilliams320) [inline libs/markdown-navigator/src/markdown-navigator-window-service/markdown-navigator-window.service.ts:45]: this should be 16px

**Review by owilliams320 — CHANGES_REQUESTED** [review]

File: libs/markdown-navigator/src/markdown-navigator-window-service/markdown-navigator-window.service.ts (hunk 1 above)
Comment (Msubhashini1) [reply libs/markdown-navigator/src/markdown-navigator-window-service/markdown-navigator-window.service.ts]: changed the pixel  to 16 px
Screen Shot 2022-05-24 at 1 33 27 PM
Screen Shot 2022-05-24 at 1 33 39 PM

**Comment by adamnel** [comment]
For the minimized state:

image

Please remove the border radius along the bottom two corners

**Comment by Msubhashini1** [comment]
> For the minimized state:
> 
> image
> 
> Please remove the border radius along the bottom two corners
Screen Shot 2022-05-26 at 12 46 51 PM

**Comment by adamnel** [comment]
Close, but this removed the radius from the top corners as well and we still want those. Here's a mock

image

**Comment by Msubhashini1** [comment]
> Close, but this removed the radius from the top corners as well and we still want those. Here's a mock
> 
Screen Shot 2022-05-26 at 1 37 30 PM

> image

**Pull request closed without merging**